asdf/asdf.py

```diff
@@ -69,6 +69,7 @@ def __init__(
             if possible and if created from `asdf.AsdfFile.open`.
 
         extensions : object, optional
+            DEPRECATED
```

Comment:
Is there a concise link we can add here that will render in the docs, something like (I'm not sure if this will render right so the docs should be checked if this suggestion is used).
```suggestion
            DEPRECATED See :ref:`extending_extensions_installing_asdf_config`
```